### ISSUE TITLE
Report - Server media storage usage per user

### ISSUE THREAD

**Issue by JacquelineMorrissette** [kobotoolbox/kpi#3067]
## Description
Create a report that can be used to verify the amount of media storage used per user account.
- This should be a downloadable report via Django admin
- This should be viewable in the user's profile in the admin UI
- The file should be updated daily

## Suggestions
- Use server logs or create a script for querying S3

**Comment by tinok** [comment]
Updated first line to be more generic. 

@jnm already has code for this that could be linked here. 

The ultimate goal is also to make this number available to each user account via an API endpoint (and to display it in the FE).

**Comment by jnm** [comment]
The code change I made was small: https://github.com/kobotoolbox/kobocat/pull/653, but (more significantly) I back-filled all the values for existing attachments by parsing the output of AWS S3 inventory.

The KoBoCAT `logger.Attachment` model should now have a `media_file_size` for every instance, except instances that don't actually have anything stored in S3. What I did not do was aggregate these per project and per user.

The code in KoBoCAT that updates the submission counters per per project and per user could be used as an example for this work.
